pkgs/development/tools/boost-build/default.nix

```diff
@@ -1,11 +1,14 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
-  name = "boost-build-2.0-m12";
-
-  src = fetchurl {
-    url = "mirror://sourceforge/boost/${name}.tar.bz2";
-    sha256 = "10sbbkx2752r4i1yshyp47nw29lyi1p34sy6hj7ivvnddiliayca";
+  name = "boost-build-2.0-${version}";
```

Comment:
I am a little bit confused about the version schema. This tags seems newer: https://github.com/boostorg/build/releases/tag/boost-1.64.0